sphinx/util/fileutil.py

```diff
@@ -35,14 +50,13 @@ def copy_asset_file(source: str, destination: str,
         # Use source filename if destination points a directory
         destination = os.path.join(destination, os.path.basename(source))
 
-    if source.lower().endswith('_t') and context is not None:
+    if _asset_destination(source) != source and context is not None:
```

Comment:
A self-nitpick: `destination` is an overloaded variable/method name here.  It should be possible (and may be safer) to improve the clarity of what is being referred to.